test/addons/external-snapshotter/controller/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 resources:
-  - https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/refs/tags/v8.1.0/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml
-  - https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/refs/tags/v8.1.0/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml
+  - https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/refs/tags/v8.2.0/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml
+  - https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/refs/heads/release-8.2/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml
```

Comment:
Lets use only rlease-8.2 or tag/v8.2.0, and open an issue about this in the repo.

Reply:
using release-8.2. [issue](https://github.com/kubernetes-csi/external-snapshotter/issues/1270)